src/Controller/Component/PrgComponent.php

```diff
@@ -2,26 +2,70 @@
 namespace Search\Controller\Component;
 
 use Cake\Controller\Component;
-use Cake\Event\Event;
 
 class PrgComponent extends Component
 {
+
+    /**
+     * Default config
```

Comment:
If it applies to all actions, `edit` and `add` actions using post will have their form data removed, which I think will be confusing for beginners.